README.md

```diff
@@ -108,6 +127,62 @@ Note: If you can’t trust your backend, some runtime validation can be useful t
 The only thing you need to manage is the configuration.
 Everything is typed and self-documented, but just in case, you can find here example configuration below:
 
+
+### Config Options
```

Comment:
I’m not sure about this section, it’s already documented in the API. Also, I’m not sure those options are not meant to be standard, this can change on every generator 😉

Reply:
ok i removed it